content/_data/changelogs/weekly.yml

```diff
@@ -4320,6 +4320,12 @@
       pull: 3999
       message: |-
         Re-enable Stapler request dispatching telemetry.
+    - type: rfe
+      issue: 47896
+      pull: 3980
+      message: |-
+        Ensure that Remoting objects are being serialized only through Remoting channels.
+        Classes like <code>hudson.FilePath</code> will not longer be serialized to the disk.
```

Comment:
```suggestion
        Certain classes which were never designed for persistence in XML will no longer be serialized to configuration files:
        <code>FilePath</code>, <code>[Stream]TaskListener</code>, and <code>ProcessTree</code>.
```

Reply:
Is `build.xml` a configuration file?

Reply:
To clarify, I would argue that users would consider it not configuration. What's wrong with just saying XML, besides a handful of plugins doing their own thing?

Reply:
Amended correspondingly.